docs/concepts/models/models.md

```diff
@@ -1,13 +1,69 @@
 ---
 title: Models
-id: model
+id: Model
 description: SQL Models
 sidebar_position: 5
 tags:
   - concepts
-  - models
+  - Models
   - sql
   - target
 ---
 
-Detailed documentation on **SQL Models** coming soon!
+**Data Modeling** refers to the shaping of your data from the raw state all the way to a transformed final state. Data engineers are typically responsible for building tables that represent source data, transforming the data and saving as intermediate tables/views, and building final tables that can be queried by a BI tool and drive decision making for an organization.
+
+Typically business logic is stored as SQL files.
+
+````mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+
+<TabItem value="py" label="myFile.sql">
+
+```
+SELECT column1, column2, ...
+FROM table_name;
+```
+
+</TabItem>
+</Tabs>
+
+````
+
+But defining the business logic in a SQL file is only the first step. Data-ready teams know that data Modeling with ad-hoc SQL statements is error-prone. How are the SQL files stored? How can the relationships between SQL files be understood? Or the relationships between tables? How is this logic shared? Can the business logic evolve as many team members contribute?
+
+**Teams shouldn't have to puzzle through storing, sharing, and understanding which tables rely on which others. Business logic should be reusable and referenceable in subsequent work.**
+
+In Prophecy and dbt, **Data Models** are SQL statements that build a single table or view - and allow for better management.
+
+Data models incorporate the step-by-step logic to transform raw data to some intermediate or final state. Each Model, stored as a `.sql` file on Git, is managed as software - with best practices like peer review and version control. The Model can include Common Table Expressions (CTEs) and [refer](https://docs.getdbt.com/docs/build/sql-models#building-dependencies-between-models) to other Models. Importantly, SQL statements with Prophecy and dbt are re-usable. When a Model is updated, any reference to that Model is likewise updated.
+
+Here we explore how to use Models in Prophecy, adopting the concept and vernacular from dbt Core™. Later, we'll see how to import dbt projects to Prophecy, or you can create a new Project and Models using Prophecy's drag-and-drop interface. After you've read this page, get hands on with Models in this getting-started [guide](/docs/getting-started/getting-started-with-low-code-sql.md#44-Develop-your-first-model).
+
+### Using Models in Prophecy
```

Comment:
Should we add a section for Macros as well?

Reply:
Yes, this will be part of more documentation planned for the low-code-sql docs section.